environment.yml

```diff
@@ -35,6 +35,7 @@ dependencies:
   - hypothesis>=6.84.0
   - gcsfs>=2022.11.0
   - ipython
+  - pickleshare  # Needed for ipython>=8.17 GH#60429
```

Comment:
This isn't needed for ipython - quite the contrary, we're experiencing this issue precisely because ipython doesn't need it! Rather, it's the IPython Sphinx directive in the docs that needs it.